gridstatus/ercot.py

```diff
@@ -1773,6 +1777,50 @@ def _make_bid_curve(df):
 
         return self.parse_doc(df, verbose=verbose)
 
+    @support_date_range("DAY_START")
+    def get_sced_system_lambda(self, date, verbose=False):
+        # ANCHOR - SCED System Lambda
+        """Get System lambda of each successful SCED
+
+        5 Minute Publish Interval
+
+        https://github.com/kmax12/gridstatus/issues/269
+
+        Arguments:
+            date (str, datetime): date to get data for
+            verbose (bool, optional): print verbose output. Defaults to False.
+
+        Returns:
+            pandas.DataFrame: A DataFrame
+
+        """
+        if date == "latest":
+            doc = self._get_document(
+                report_type_id=SCED_SYSTEM_LAMBDA_RTID,
+                date=date,
+                verbose=verbose,
+            )
+            df = pd.read_csv(doc.url, compression="zip")
+
+            return df
+        else:
+            doc = self._get_documents(
+                report_type_id=SCED_SYSTEM_LAMBDA_RTID,
+                date=date,
+                verbose=verbose,
+                extension="csv",
+            )
+            df = pd.DataFrame()
```

Comment:
small thing, it's more efficient to rewrite this do just one concat

```python
dfs = []
for doc in docs:
    log(f"Reading {doc.url}", verbose)
    new_df = pd.read_csv(doc.url, compression="zip")
    dfs.append(new_df)
df = pd.concat(dfs)
```

Reply:
nice catch, fixed